pull_request_template.md

```diff
@@ -0,0 +1,19 @@
+# LAN-B PR Template 
+	Trello Card URL : 
+
+# Description of changes: 
+
+Please enter a description of what has changed in your code.  Also include the reason for the changes, and the type of change. 
+
+## How was it tested? 
+
+include how you tested your software, and explain steps taken to verify it meets LAN-B's standard practices. 
+
+## PR Checklist: 
```

Comment:
I think the checklist having checkable boxes is worthwhile to make it a more active exercise for the PR author; should be able to have those with spaced out brackets, like `[ ]`

Reply:
Will make the checklist update - for my "its documenting" was a description of my own pull request ... Not whats on the template.  if you look on the template it says 

## How was it tested? 

include how you tested your software, and explain steps taken to verify it meets LAN-B's standard practices.

Reply:
Sorry if I wasn't clear, I know that! Grammatically and descriptively it is not up to the standard we want to set with these descriptions, so I think it could use another pass.